mystery_o_matic/output/latex/__init__.py

```
@@ -0,0 +1,148 @@
+            sub_bullets.append(clue)
+
+        final_locations_bullets = mystery.final_locations_intro[language]
+        final_locations_bullets += get_bullet_list(sub_bullets)
+        bullets.append(final_locations_bullets)
+        bullets.append(NoOneElseStatement().string()[language])
+
+        initial_clues = get_bullet_list(bullets)
+
+        additional_clues = []
+
+        for i, clue in enumerate(mystery.additional_clues):
```

Ruff

Loop control variable `i` not used within loop body

(B007)